src/BrightScriptDebugSession.spec.ts

```diff
@@ -415,6 +416,116 @@ describe('Debugger', () => {
             await doTest('function   Main()\nend function', 'function   Main()', 1);
             await doTest('function Main   ()\nend function', 'function Main   ()', 1);
         });
+
+        it('works for sub RunScreenSaver', async () => {
+            await doTest('sub RunScreenSaver()\nend sub', 'sub RunScreenSaver()', 1);
+            //works with extra spacing
+            await doTest('sub   RunScreenSaver()\nend sub', 'sub   RunScreenSaver()', 1);
+            await doTest('sub RunScreenSaver   ()\nend sub', 'sub RunScreenSaver   ()', 1);
+        });
+
+        it('works for function RunScreenSaver', async () => {
+            await doTest('function RunScreenSaver()\nend function', 'function RunScreenSaver()', 1);
+            //works with extra spacing
+            await doTest('function   RunScreenSaver()\nend function', 'function   RunScreenSaver()', 1);
+            await doTest('function RunScreenSaver   ()\nend function', 'function RunScreenSaver   ()', 1);
+        });
+    });
+
+    describe('injectRaleTrackerTaskCode', () => {
+        let key: string;
+        let trackerTaskCode: string;
+        let folder;
+
+        beforeEach(() => {
+            key = 'vscode_rale_tracker_entry';
+            trackerTaskCode = `if true = CreateObject("roAppInfo").IsDev() then m.vs_code_tracker_task = createObject("roSGNode", "TrackerTask") ' Roku Advanced Layout Editor Support`;
```

Comment:
Your test has the old `vs_code_tracker_task` text, but it's not failing the test. Is that a bug in the test?

Reply:
No, what you'er looking at is the code that is going to be injected into the channel in the place of the comment. It's creating a `m.vs_code_tracker_task` variable to store the tracker in. I can change this variable to `m.vscode_rale_tracker_task` if you would like.

Reply:
ah. ok, that makes more sense. Yeah, let's standardize it to `vscode` anywhere that `vs_code` is used.

Reply:
Done.